README.md

```diff
@@ -29,7 +29,7 @@ You'll have a new folder in the directory called `_site` that has all of the com
 
 ## Deployment
 
-For deploying to S3, we use a tool called `s3_website` which looks at a file named `s3_website.yml` for configuration information. Take a look at the example file, add you S3 API keys and bucket name, and save the file as `s3_website.yml`
+For deploying to S3, we use a tool called `s3_website` which looks at a file named `s3_website.yml` for configuration information. Take a look at the example file, add you S3 API keys and bucket name and save the file as `s3_website.yml`
```

Comment:
@Cbeck527 Perhaps we can compromise and add the comma back here and not in `_config.yml`? I can see it making sense here for clarity, given the "and" in the previous clause.